mizar/dp/mizar/workflows/builtins/networkpolicies/create.py

```diff
@@ -76,12 +76,22 @@ def handle_networkpolicy_create_update(self, name, pod_label_dict, policy_types)
 
     def generate_data_for_networkpolicy_ingress(self, ep):
         data = self.init_data_for_networkpolicy()
-        #TODO build ingress data
+        gress_type = "ingress"
```

Comment:
Can you rename this to "direction" or something else?

Reply:
Fixed